docs/overview/office-add-ins-fundamentals.md

```diff
@@ -0,0 +1,164 @@
+
+![An image that shows add-in commands on the ribbon, a task pane, and a dialog box in an Office document](../images/add-in-ui-elements.png)
+
+For more information about extending the Office UI, see [Office UI elements for Office Add-ins](../design/interface-elements.md).
+
+## Core development concepts 
+
+An Office Add-in consists of two parts:
+
+- The add-in manifest (an XML file) that defines the settings and capabilities of the add-in.
+
+- The web application that defines the UI and functionality of add-in components such as task panes, content add-ins, and dialog boxes.
+
+The web application uses the Office JavaScript API to interact with content in the Office document where the add-in is running. Your add-in can also do other things that web applications typically do, like call external web services, facilitate user authentication, and more.
+
+### Defining an add-in's settings and capabilities
+
+An Office Add-in's manifest (an XML file) defines the settings and capabilities of the add-in. You'll configure the manifest to specify things such as:
+
+- Metadata that describes the add-in (for example, ID, version, description, display name, default locale).
+- Office applications where the add-in will run.
+- Permissions that the add-in requires.
+- How the add-in integrates with Office, including any custom UI that the add-in creates (for example, custom tabs, ribbon buttons).
+- Location of images that the add-in uses for branding and command iconography.
+- Dimensions of the add-in (for example, dimensions for content add-ins, requested height for Outlook add-ins).
+- Rules that specify when the add-in activates in the context of a message or appointment (for Outlook add-ins only).
+
+For detailed information about the manifest, see [Office Add-ins XML manifest](add-in-manifests.md).
+
+### Interacting with content in an Office document
+
+An Office Add-in can use the Office JavaScript APIs to interact with content in the Office document where the add-in is running. 
+
+#### Accessing the Office JavaScript library
+
+The Office JavaScript library can be accessed via the Office JS content delivery network (CDN) at: `https://appsforoffice.microsoft.com/lib/1/hosted/Office.js`. To use Office JavaScript APIs within any of your add-in's web pages, you must reference the CDN in a `<script>` tag in the `<head>` tag of the page.
+
+```html
+<head>
+    ...
+    <script src="https://appsforoffice.microsoft.com/lib/1/hosted/Office.js" type="text/javascript"></script>
+</head>
+```
+
+> [!NOTE]
+> To use preview APIs, reference the preview version of the Office JavaScript library on the CDN: https://appsforoffice.microsoft.com/lib/beta/hosted/office.js.
+
+For more information about accessing the Office JavaScript library, including how to get IntelliSense, see [Referencing the JavaScript API for Office library from its content delivery network (CDN)](../develop/referencing-the-javascript-api-for-office-library-from-its-cdn.md).
```

Comment:
The title of the linked article is a bit misleading since you're not getting the types from the CDN; you're getting them from Definitely Typed. Also, that article seems off. There's a reference to "Excel-15.js", which I haven't heard of.

Reply:
Good catch. Beyond the scope of this PR, but I'll log this as a new work item in VSO, to be addressed when we update/improve content specifically in the **Develop** section.